scripts/change-log-generator.js

```diff
@@ -54,19 +54,51 @@ const typesToIgnore = [
   'revert'
 ];
 
+const RELEASE_OVERRIDE_ARG = '-o';
+const RELEASE_DATE_ARG = '-t';
+const VERBOSE_LOGGING_ARG = '-v';
+const PACKAGES_TO_IGNORE_ARG = '-i';
+
+function getArgumentValue(arg) {
+  const argIndex = process.argv.indexOf(arg);
+  if (argIndex > -1) {
+    const argValue = process.argv[argIndex + 1];
+    return argValue && !argValue.startsWith('-') ? argValue : '';
```

Comment:
This is an important check since the VSCode task will still supply the argument flag, even if the argument value is blank. So it's possible for argValue at this point in time to be the next flag, not an actual arg value.